tests/deprecated_api/test_remove_1-8.py

```diff
@@ -688,3 +689,38 @@ def on_load_checkpoint(self, checkpoint):
         " v1.6 and will be removed in v1.8. Use `load_state_dict` instead."
     ):
         trainer.fit(model)
+
+
+@pytest.mark.parametrize(
```

Comment:
I don't think all of these are required here since they are already tested in other files. Just take one of these cases and check the deprecated_call as you have already done at the end.